composer/datasets/in_context_learning_evaluation.py

```diff
@@ -44,6 +45,13 @@ def strip_data(example: Dict) -> Dict:
     Returns:
         dict: The same dictionary with .strip() applied to any value in the dict that is a string
     """
+    warnings.warn(
```

Comment:
you don't need to explicitly deprecate all of these helper functions. can just do the classes and the builders